### PR TITLE
feat: `bindings-expand` Earthly target

### PR DESCRIPTION
**Pull request by Mr-Leshiy**
# Description

Added new `bindings-expand` Earthly target, which expands `wasmtime::bindgen!` into the separate file

## Related Pull Requests

Depends on https://github.com/input-output-hk/catalyst-ci/pull/190

### CODE REVIEW

File: hermes/Earthfile [hunk 1]
```diff
@@ -13,6 +13,13 @@ builder:
     RUN mkdir /wasm
     COPY --dir ../wasm+wasi-src/wasi /wasm/wasi
 
+# Expands `wasmtime::bindgen!` macro into the `bindings.rs` file
+bindings-expand:
+    FROM +builder
+
+    RUN cargo expand -p hermes --lib runtime_extensions::bindings > bindings.rs
+    SAVE ARTIFACT bindings.rs AS LOCAL bindings.rs
+
```
Comment (FelipeRosa) [inline hermes/Earthfile:22]: I would suggest that this is a resposibility of whoever changes the bindings, WDYT?

Comment (Mr-Leshiy) [reply hermes/Earthfile:22]: Did not get what do you mean.
I have added this just for the convenience of the local development, so you can alway easily take a look how it is expanded

**Review by bkioshn — COMMENTED** [review]
Looks good 🙌🏼

**Review by apskhem — COMMENTED** [review]
Looks fine.

**Review by FelipeRosa — APPROVED** [review]
LGTM

**Pull request merged**